lib/components/InlinePopup/InlinePopup.module.scss

```diff
@@ -62,9 +60,9 @@ $swatches-horizontal: 3;
         z-index: $z-index-flyouts-inline-btn;
         padding: $grid-size / 2;
         margin-top: -1px; // pulls the inline popup panel up and underneath the container
-        background-color: var(--color-bg-panel);
-        border: 1px solid var(--color-border-rest);
-        box-shadow: var(--shadow-popup-panel);
+        background-color: var(--color-content-background-secondary);
+        border: 1px solid var(--color-foreground-secondary);
+        box-shadow: -3px 3px 10px -3px var(--color-dark-transparent-primary);

```

Comment:
lol another custom box shadow :)

Reply:
same thing as the others... it was the exact value we had in `shadow-popup-panel`